_config.yml

```diff
@@ -43,4 +43,22 @@ includes_dir: .
 
 plugins:
   - jekyll-last-modified-at
-  - jekyll-redirect-from
\ No newline at end of file
+  - jekyll-redirect-from
+  - jekyll-toc
```

Comment:
Can you check if this plugin is allowed in GitHub pages?

Reply:
Yeah, it is. I checked before using it...